x/bep3/types/swap.go

```diff
@@ -264,3 +264,42 @@ func (direction SwapDirection) IsValid() bool {
 	}
 	return false
 }
+
+type AugmentedAtomicSwap struct {
```

Comment:
I matched this naming with `AugmentedCDP` with idea that one pattern is better than two patterns of decorating output.

Alternatively, this could go be named `QueryAtomicSwapResponse` in `types/querier.go`.